CMakeLists.txt

```
@@ -133,7 +133,7 @@ add_library(${PACKAGE} ${lib_SRC})
 target_include_directories(${PACKAGE}
   INTERFACE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-    $<INSTALL_INTERFACE:include>
+    $<INSTALL_INTERFACE:${includedir}>
```

No, really:

```suggestion
    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

the relative path, without pkg-configs `${prefix}/`.

---

Actually I tested both cases manually successfully.

---

Opting for @dg0yt proposal as it is used by GDAL and PROJ